hack/jenkins/e2e.sh

```diff
@@ -101,6 +101,9 @@ GCE_FLAKY_TESTS=(
     "DaemonRestart"
     "ResourceUsage"
     "monotonically\sincreasing\srestart\scount"
+    "then\sschedule\sit\son\sanother\shost" # pd.go
+    "should\sbe\sable\sto\schange\sthe\stype\sand\snodeport\ssettings\sof\sa\sservice" # service.go
+    "allows\sscheduling\sof\spods\son\sa\sminion\safter\sit\srejoins\sthe\scluster" # resize_nodes.go
```

Comment:
Can you add issue number to each of the bugs? These are respectively:
#13257 #13032 #13258